src/components/refactor/utils/yup-config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import * as yup from 'yup';
+
+yup.setLocale({
+    mixed: {
```

Comment:
what is the strategy for the other messages ? add them as we see them ? https://github.com/jquense/yup/blob/d072af33b84acef4f2bd355ff5637048edb95bae/src/locale.ts#L66